lib/rialto/etl/writers/sparql_statement_writer.rb

```diff
@@ -190,6 +196,33 @@ def advisees_to_statements(advisor, advisor_id, hash, graph_name)
         end
         # rubocop:enable Metrics/AbcSize
         # rubocop:enable Metrics/MethodLength
+
+        # rubocop:disable Metrics/MethodLength
+        # rubocop:disable Metrics/AbcSize
+        def positions_to_statements(subject, subject_id, hash, graph_name)
+          statements = []
+          hash['@positions'].each do |position_hash|
+            # statements << serialize_hash(position_hash, graph_name)
```

Comment:
Remove?

Reply:
Yes.

Reply:
@jcoyne care to review this PR once more?